creator-node/src/components/replicaSet/replicaSetController.js

```diff
@@ -175,7 +192,7 @@ const _syncRouteController = async (req, res) => {
     )
   }
 
-  return successResponse()
+  return successResponse({ syncUuid })
```

Comment:
whats benefit to returning syncUuid for an immediate sync?

Reply:
immediate and non-immediate syncs are both processed by the issueSyncRequest job processor so they need the syncUuid to perform the same polling logic

Reply:
but when immediate sync returns the process has already completed so there is no further polling required? the response from the route tells us everything we need, right?

Reply:
fair, that is out of scope here but weird
can you throw linear card